src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -56,8 +56,9 @@ class OpenEphysLegacyRecordingExtractor(NeoBaseRecordingExtractor):
         If there are several blocks (experiments), specify the block index you want to load
     all_annotations: bool, default: False
         Load exhaustively all annotation from neo
-    ignore_timestamps_errors: bool, default: False
-        Ignore the discontinuous timestamps errors in neo
+    ignore_timestamps_errors: None
+        Deprecated kwarg. This is now ignored.
+        neo.OpenEphysRawIO is now handling gaps directly but make the read slower.
```

Comment:
```suggestion
        Deprecated keyword argument. This is now ignored.
        neo.OpenEphysRawIO is now handling gaps directly but makes the read slower.
```

Could I request that we change kwarg here to keyword argument just to be explicit in the docstring description. Considering we have some Python beginners and saying args and kwargs can be a bit confusing I think explicit in the docstring is beneficial. Also corrected one tiny typo.